engine/tests/mechanics/games/test_credentials_derive.py

```
@@ -0,0 +1,196 @@
+    CredentialsGame,
+    DEFAULT_RESTRICTIONS,
+    Indication,
+    Region,
+    Restrictions,
+    RestrictionLevel,
+    derive_disposition,
+)
+
+D = CredentialDisposition
+S = CredentialStatus
+I = Indication
```

Ruff (0.15.13)

[error] Ambiguous variable name: `I`

(E741)